src/main/java/com/exactpro/th2/fix/client/fixBean/FixBean.java

```diff
@@ -1,33 +1,49 @@
 package com.exactpro.th2.fix.client.fixBean;
 
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import org.apache.commons.lang3.builder.ToStringBuilder;
 import org.apache.commons.lang3.builder.ToStringStyle;
 
+import static com.exactpro.th2.fix.client.util.FixBeanUtil.addToConfig;
+
 public class FixBean {
 
-    protected String applicationID = "client";
+    @JsonProperty(required = true)
     protected String fileStorePath = "storage/messages/";
+    @JsonProperty(required = true)
     protected String fileLogPath = "outgoing";
+    @JsonIgnore
     protected String connectionType = "initiator";
+    @JsonProperty(required = true)
     protected long reconnectInterval = 60;
+    @JsonProperty(required = true)
     protected long heartBtInt = 30;
+    @JsonIgnore
     protected String useDataDictionary = "Y";
     protected String validateUserDefinedFields = "N";
     protected String validateIncomingMessage = "N";
     protected String refreshOnLogon = "Y";
+    @JsonProperty(required = true)
     protected String nonStopSession = "Y";
```

Comment:
Same as above

Reply:
these are all required fields, the app doesn't run without them.

Reply:
Yeah, but there's no need to require a user to set them - we can use default bean-field value instead